.github/workflows/_build-pdffit2-package.yml

```diff
@@ -49,7 +49,7 @@ jobs:
         with:
           python-version: ${{ matrix.python[0] }}
 
-      - name: Build wheels for Linux
+      - name: Linux build wheels
```

Comment:
Start with "Linux" or "macOS"  so that using the first word, we can focus on the OS of interest.